### ISSUE TITLE
Package naming

### ISSUE THREAD

**Issue by mathieu1** [conda-forge/pdal-feedstock#21]
From https://github.com/PDAL/python/issues/1 :

> The conda package includes the Python plugin (filters.python) but not the Python package (import pdal) (this is a work in progress that I would expect to be released in the coming days)

Do you plan on putting both the python plugin and the python package in one conda package or in separate ones?
- if they're both in the same conda package (naturally called `pdal`), then it's unclear what the version should be (current versions are 1.7.1 and 2.0.0). 
- if they're in different packages, how will they be named? Taking `gdal` as a point of comparison, the core package is called `libgdal` while `gdal` is for the python wrapper.

Side remark: the current package only needs to depend on `libgdal`.

**Comment by chambbj** [comment]
> Side remark: the current package only needs to depend on libgdal.

Correct. Fixed at https://github.com/conda-forge/pdal-feedstock/commit/86c020af3fe871e56d6d2b7776c997b61f3615e2.

> Do you plan on putting both the python plugin and the python package in one conda package or in separate ones?

The plan is to move all Python related features to the PDAL/python repo. After some discussion with the conda-forge guys a while back, we decided that `python-pdal` would be an appropriate package name. I had pushed back somewhat on calling the core PDAL package `libpdal` as it's not just the library, but also the application. Folks looking specifically for the PDAL application might not expect to find it under `libpdal`, and while they would've gotten it via installation of `pdal` it would have been as a dependency and quite by accident. PDAL/python will follow it's own release schedule and versioning.
